### PR TITLE
Activate travis

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
+
+dist: trusty
+
+services:
+  - postgresql
```
Comment (sbrunner) [inline .travis.yml]: You add the service and remove the packages after ..., is it relay useful?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
```
Comment (sbrunner) [inline .travis.yml]: The project is only working with python 2.7?

File: .travis.yml (hunk 1 above)
Comment (arnaud-morvan) [inline .travis.yml]: You're right, this is not really useful. With service postgresql, I get errors : 
```
ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
```
So I remove it, reinstall, and all works fine.

File: .travis.yml (hunk 2 above)
Comment (arnaud-morvan) [inline .travis.yml]: I've added python 3.4, and travis is still happy.

File: .travis.yml (hunk 1 above)
Comment (sbrunner) [inline .travis.yml]: :-)

File: .travis.yml (hunk 2 above)
Comment (sbrunner) [inline .travis.yml]: Thanks :-)
By the way I don't know if the python 2.7 is steel needed ...

**Comment by fvanderbiest** [comment]
Improved travis file on branch https://github.com/camptocamp/c2cgeoform/tree/travis-on-docker : runs on container infrastructure.

**Comment by fvanderbiest** [comment]
Using container infra fails here ...  I reported it on https://github.com/travis-ci/travis-ci/issues/8292

**Comment by fvanderbiest** [comment]
Finally I got it working on branch https://github.com/camptocamp/c2cgeoform/tree/travis-on-docker
Should I do another PR superseding this one ?

**Comment by arnaud-morvan** [comment]
Thanks, update my PR or create a new one, but I think you should squash the 3 commits about travis file.

**Comment by fvanderbiest** [comment]
My commits squashed... ready to go.

**Pull request merged**